CHANGES.md

```diff
@@ -15,7 +15,9 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+- deploy-data: allow more flexibility to deploy files from other checkout in same config file
```

Comment:
Please improve the description to be more explicit, such as mentioning `SRC_DIR` value and the obtained behavior. Looking at this later on will be hard to understand.

Reply:
Indeed, I could re-phrase the code comment in this changelog as well.